src/service/logging.cpp

```diff
@@ -80,13 +82,13 @@ logging_t::logging_t(context_t& context, asio::io_service& asio, const std::stri
             registry.add<blackhole::sink::socket::udp_t>();
 
             std::stringstream stream;
-            stream << boost::lexical_cast<std::string>(context.config.logging.loggers);
+            stream << boost::lexical_cast<std::string>(context.config().logging.loggers);
 
             auto log = registry.builder<blackhole::config::json_t>(stream)
                 .build(backend);
 
             log.filter([&](const blackhole::record_t& record) -> bool {
-                return record.severity() >= context.config.logging.severity || !trace_t::current().empty();
+                return record.severity() >= context.config().logging.severity || !trace_t::current().empty();
```

Comment:
I doubt that someone will ever notice this one more call (which can be and probably will be inlined during lto) on the background of std::function call, but ok, I'll fix